Dockerfile

```diff
@@ -17,6 +17,9 @@ RUN apt-get update
 # Any text editor
 RUN apt-get install -y nano net-tools
 
+# Extra tools
+RUN npm install -g bunyan
```

Comment:
You can merge this run with this one: 

#npm packages
RUN npm install -g bunyan pm2 standard